### PR TITLE
Added e2e workflow

### PR DESCRIPTION
**Pull request by spau6**
Added e2e workflow

### CODE REVIEW

**Review by amar-c8y — DISMISSED** [review]

File: .github/workflows/notify-ms-teams/action.yml
Comment (ghost) [inline .github/workflows/notify-ms-teams/action.yml:1]: remove ms notifications

File: .github/workflows/widget-cypress-test.yml [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+      spec_files:
+        description: "Spec files(comma separated list)"
+        required: false
+        type: string
+      browser:
+        description: "Browser"
+        required: true
+        type: choice
+        default: "chrome"
+        options:
```
Comment (ghost) [inline .github/workflows/widget-cypress-test.yml:19]: ensure all these options work if provided

File: .github/workflows/widget-cypress-test.yml [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+      tenantUrl: ${{ inputs.tenantUrl }}
+      project_directory: cypress-E2E-tests
+      spec_files: ${{needs.setup-variables.outputs.spec_files}}
+      browser: ${{github.event.inputs.browser}}
+
+    secrets:
+      AUTOTEST_USERNAME1: ${{ secrets.AUTOTEST_USERNAME1 }}
+      AUTOTEST_USERNAME1_PASSWORD: ${{ secrets.AUTOTEST_USERNAME1_PASSWORD }}
+      AUTOTEST_USERNAME2: ${{ secrets.AUTOTEST_USERNAME2 }}
+      AUTOTEST_USERNAME2_PASSWORD: ${{ secrets.AUTOTEST_USERNAME2_PASSWORD }}
```
Comment (ghost) [inline .github/workflows/widget-cypress-test.yml]: remove unused placeholders

File: cypress-E2E-tests/cypress/integration/integration.cy.ts [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+  // it('TC_Asset_Properties_Widget_Integration_002', () => {
+  //   const instance1Title = 'Asset Properties';
+  //   const instance2Title = 'Test';
+  //   const propFeildElement = '#formly_2_string_color_0';
+  //   const saveElement = '.btn-primary';
+  //   const assetName = 'Test Asset5';
+  //   const titleFieldId = '#widgetTitle';
+  //   cy.selectAssetPropertyAndSave(assetName, propKey);
+  //   cy.get(asset_properties_widget_elements.widgetDashboardAddWidgetButton).click();
+  //   cy.get(asset_properties_widget_elements.cardElement).click();
```
Comment (ghost) [inline cypress-E2E-tests/cypress/integration/integration.cy.ts]: remove commented code

**Review by ghost — CHANGES_REQUESTED** [review]

File: .github/workflows/widget-cypress-test.yml (hunk 2 above)
Comment (spau6) [reply .github/workflows/widget-cypress-test.yml]: They are used ones

File: .github/workflows/notify-ms-teams/action.yml
Comment (spau6) [reply .github/workflows/notify-ms-teams/action.yml:1]: removed

File: cypress-E2E-tests/cypress/integration/integration.cy.ts (hunk 3 above)
Comment (spau6) [reply cypress-E2E-tests/cypress/integration/integration.cy.ts]: they were not added

File: .github/workflows/widget-ci.yml [hunk 4]
```diff
@@ -14,3 +14,12 @@ jobs:
     uses: ./.github/workflows/unit-test-widget.yml
     with:
       test_command: "yarn test"
+
+  build-webapp:
+    uses: ./.github/workflows/build-webapp.yml
+    with:
+      package_name: "asset-properties-widget"
```
Comment (ghost) [inline .github/workflows/widget-ci.yml]: please verify if the build created with this as package name works as expected.

**Review by ghost — CHANGES_REQUESTED** [review]

File: .github/workflows/widget-ci.yml (hunk 4 above)
Comment (spau6) [reply .github/workflows/widget-ci.yml]: I have removed the package name as it is not required for us

File: .github/workflows/widget-cypress-test.yml (hunk 1 above)
Comment (spau6) [reply .github/workflows/widget-cypress-test.yml:19]: yes it is supported in all 3 browsers

**Comment by spau6** [comment]
Widget CI run: https://github.com/SoftwareAG/cumulocity-asset-properties-widget/actions/runs/6536132148
Widget E2E run: https://github.com/SoftwareAG/cumulocity-asset-properties-widget/actions/runs/6544985622

**Review by ghost — APPROVED** [review]

**Review by amar-c8y — APPROVED** [review]

**Pull request merged**